src/WebViewTypes.ts

```diff
@@ -10,6 +10,7 @@ import {
   UIManagerStatic,
   NativeScrollEvent,
 } from 'react-native';
+import type { DirectEventHandler } from 'react-native/Libraries/Types/CodegenTypes';
```

Comment:
for some reason this import can't be found, but it works when integration in platform

Reply:
Bump react native 👍 
https://github.com/Ldoppea/react-native-webview/blob/667cfec3900560a724c4ad111d4bdbb76f7eba14/package.json#L63